compiler/src/iree/compiler/Pipelines/Options.h

```diff
@@ -121,7 +121,9 @@ struct GlobalOptimizationOptions {
   // Enables transposing all concatenations to the outer most dimension.
   bool outerDimConcat = false;
 
-  // Enables data tiling.
+  // Enables data tiling in global optimization phase.There are two data-tiling
+  // flags during the transition state. The other has to be off if it is
```

Comment:
```suggestion
  // flags during the transition state. The other has to be off if this one is
```
For clarity, this is what you meant right?

Reply:
Yes, it is clearer. Thanks!